### PR TITLE
travis-ci enable tests under address sanitizier

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

File: cmake/common/sanitizers.cmake [hunk 1]
```diff
@@ -36,6 +36,16 @@ option(SANITIZE_MEMORY "Enable MemorySanitizer" Off)
 option(SANITIZE_THREAD "Enable ThreadSanitizer" Off)
 option(SANITIZE_UNDEFINED "Enable UndefinedBehaviorSanitizer" Off)
 
+if(DEFINED ENV{PX4_ASAN})
+	set(SANITIZE_ADDRESS ON)
+elseif(DEFINED ENV{PX4_MSAN})
```
Comment (bkueng) [inline cmake/common/sanitizers.cmake]: Can you add these to the dev-guide? PX4_ASAN is already there: https://github.com/PX4/Devguide/blob/master/en/debug/simulation_debugging.md

**Comment by dagar** [comment]
No, I don't think we want these messages, at least not for small changes. This is the default for codecov and it needs to be configured.

**Comment by dagar** [comment]
image

**Comment by dagar** [comment]
image

**Comment by nicolaerosia** [comment]
@dagar does continuous-integration/sitl/pr also run with asan?

**Comment by dagar** [comment]
@nicolaerosia not yet, but that's the plan (if I can expand the CI resources).

File: cmake/common/sanitizers.cmake (hunk 1 above)
Comment (dagar) [reply cmake/common/sanitizers.cmake]: The others don't quite work yet, but I plan to get them working and added to builds.

**Comment by dagar** [comment]
Added to the builds, but allowed to fail for now until the initial issues are resolved.

**Pull request merged**